src/GoalioRememberMe/Module.php

```diff
@@ -26,12 +27,17 @@ public function getConfig() {
     public function getServiceConfig() {
         return array(
             'invokables' => array(
-                'GoalioRememberMe\Authentication\Adapter\Cookie' => 'GoalioRememberMe\Authentication\Adapter\Cookie',
                 'GoalioRememberMe\Form\Login'                    => 'GoalioRememberMe\Form\Login',
-                'goaliorememberme_rememberme_service'            => 'GoalioRememberMe\Service\RememberMe',
-            ),
+             ),
 
             'factories' => array(
+                'goaliorememberme_rememberme_service' => function ($sm) {
```

Comment:
I think the factories should move to separate file so we dion't break the caching of this.